ml.q

```diff
@@ -258,12 +258,18 @@ cgroup:{[df;X;C]value group imin f2nd[df X] C}
 / matri(X) to the nearest (C)entroid and then uses the (c)entroid (f)unction
 / to update the centroid location.
 lloyd:{[df;cf;X;C]cf X@\: cgroup[df;X;C]}
+/ uses (r)esponsibility (f)unction David Mackay's Information Theory... pg 289
```

Comment:
there is a strict 76 character limit to the code (so it can fit within the margins of the book).  this comment is 77 characters

Reply:
fixed, (I knew it has to be tight, but missed by 1 char)